chapters/planning-data-work.tex

```diff
@@ -1,147 +1,459 @@
-
-Find your \textbf{home folder}. On MacOS, this will be a folder with your username.
-On Windows, this will be something like ``This PC''. (It is never your desktop.)
-Nearly everything we talk about will assume you are starting from here.
-Ensure you know how to get the \textbf{absolute file path} for any given file.
+  \index{password protection}
+All machines should have \textbf{hard disk encryption} enabled.
+  \index{encryption}
+Disk encryption is built-in on most modern operating systems;
+the service is currently called BitLocker on Windows or FileVault on MacOS.
+Disk encryption prevents your files from ever being accessed
+without first entering the system password.
+This is different from file-level encryption,
+which makes individual files unreadable without a specific key.
+(We will address that in more detail later.)
+As with all critical passwords, your system password should be strong,
+memorable, and backed up in a separate secure location.
+
+Make sure your computer is backed up to prevent information loss.
+  \index{backup}
+Follow the \textbf{3-2-1 rule}: maintain 3 copies of all critical data,
+on at least 2 different hardware devices you have access to,
+with 1 offsite storage method.\sidenote{
+  \url{https://www.backblaze.com/blog/the-3-2-1-backup-strategy/}}
+One reasonable setup is having your primary computer,
+a local hard drive managed with a tool like Time Machine
+(alternatively, a fully synced secondary computer),
+and either a remote copy maintained by a cloud backup service
+or all original files stored on a remote server.
+Dropbox and other synced files count only as local copies and never as remote backups,
+because other users can alter or delete them.
+  \index{Dropbox}
+
+Find your \textbf{home folder}. It is never your desktop.
```

Comment:
Why do we introduce *home folder* without not using that term again?

Reply:
**This PC** is not a great equivalence to _home folder_ and we are not really talking about home folders again. I think we should remove this.

Reply:
c17d4b1e4ffbf81f2492900a274face4fda34d68